### PR TITLE
Fix overrides filename

### PR DESCRIPTION
**Pull request by KevinRansom**
F# AtrributeInfo doesn't work correctly when compiled using visual studio, because the filename for the shim is incorrect.

This change fixes that.

Change also needs to go to release/2.2.1

### CODE REVIEW

**Comment by nguerrera** [comment]
> Change also needs to go to release/2.2.1

That will happen automatically if approved for 2.1.5xx (15.9) and this is merged

**Review by nguerrera — APPROVED** [review]

**Comment by nguerrera** [comment]
@Pilchie for approval for 2.1.500 / VS 15.9

@livarcocc FYI

@KevinRansom Hold on merging until we have "approved to merge" label

**Comment by Pilchie** [comment]
Approved for 15.9 Preview *4* (P3 would require QB insertion right now)

**Comment by nguerrera** [comment]
@livarcocc @Pilchie Should we be holding merges in case we need to insert something else for P3?

**Review by nguerrera — CHANGES_REQUESTED** [review]
Putting "request changes" for now until we're sure branch is open for 15.9 P4.

**Comment by livarcocc** [comment]
@nguerrera, yes. Let's hold off until we can "open" the branch for P4.

**Comment by livarcocc** [comment]
we should be good to merge this now.

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu14.04 Debug

**Review by nguerrera — APPROVED** [review]

**Pull request merged**
